src/olympia/signing/views.py

```diff
@@ -232,15 +236,30 @@ def handle_upload(self, request, addon, version_string, guid=None):
                 ))
 
             parsed_data['guid'] = guid
+        elif not guid and package_guid:
+            guid = package_guid
 
-        if package_guid or guid:
+        if guid:
             # If we did get a guid, regardless of its source, validate it now
             # before creating anything.
-            if not amo.ADDON_GUID_PATTERN.match(package_guid or guid):
+            if not amo.ADDON_GUID_PATTERN.match(guid):
                 raise forms.ValidationError(
                     ugettext('Invalid Add-on ID in URL or package'),
                     status.HTTP_400_BAD_REQUEST)
 
+        block_qs = Block.objects.filter(guid=addon.guid if addon else guid)
+        if block_qs and block_qs.first().is_version_blocked(version_string):
+            msg = ugettext(
+                'Version {version} matches {block_url} for this add-on. '
+                'You can contact {amo_admins} for additional information.')
```

Comment:
You have different variables than the message in devhub so this would be a different string - could you change this so that we don't have 2 different translations ?

Reply:
the string is slightly different because the devhub message is html (`<a href...` for the links/email) and the api is plain text with the url directly in the content.  I'll attempt to rephrase so it works for both places.

Reply:
https://github.com/mozilla/addons-server/pull/14295/commits/aa2cc12040ec960ded4f14e1f6b0076b94f5a66e